MapboxSearch/autofill/src/main/java/com/mapbox/search/autofill/AddressAutofillResult.kt

```diff
@@ -14,6 +15,11 @@ public class AddressAutofillResult internal constructor(
      */
     public val suggestion: AddressAutofillSuggestion,
```

Comment:
I would removed suggestion from result and placed inlined all the needed for result fields here to avoid bugs like this https://github.com/mapbox/mapbox-search-android/pull/168#discussion_r1398405216.

Can be done as a follow up.

Reply:
This will be done as a follow-up since it's not the only place. It has to be addressed in other parts of the SDK too.

Reply:
Filed SSDK-587